formats/src/main/scala/flatgraph/formats/graphson/GraphSONProtocol.scala

```diff
@@ -0,0 +1,78 @@
+package flatgraph.formats.graphson
+import spray.json._
+
+object GraphSONProtocol extends DefaultJsonProtocol {
+
+  implicit object PropertyValueJsonFormat extends RootJsonFormat[PropertyValue] {
+    def write(c: PropertyValue): JsValue = {
+      c match {
+        case x: StringValue  => JsString(x.`@value`)
+        case x: BooleanValue => JsBoolean(x.`@value`)
+        case x: ListValue =>
+          JsObject("@value" -> JsArray(x.`@value`.map(write).toVector), "@type" -> JsString(x.`@type`))
+        case x: LongValue =>
+          JsObject("@value" -> JsNumber(x.`@value`), "@type" -> JsString(x.`@type`))
```

Comment:
There is a perennial issue with 64 bit integers that cannot be faithfully represented as Double (because javascript number is double). Does your code work with that?

This perennial issue does not affect our node-ids: I carefully made sure that the leading 16 bit are zero, and 48 bit integers always fit faithfully into a double ;)

Reply:
This is old copied code, and yes for sure this won't be correct, and it's funny and sad at the same time :) 
But for the use case it doesn't really matter, at least it hasn't so far.